doc/playbook.rst

```diff
@@ -67,6 +67,36 @@ Any subsequent builds of the service container will build against both bases.
 
 This is the only situation in which a change to ``BASE_TAG_LIST`` should be committed to ``main``.
 
+In case your intended Base Container is not a tagged Science Pipelines Release: Building the Science Pipelines Manually
+-----------------------------------------------------------------------------------------------------------------------
+It will sometimes be necessary to compile a container with the LSST Science Pipelines manually. Generally, this only occurs if the intended daily or weekly stack does not compile. In these cases, the Science Pipelines themselves must be built ahead of the base container. Here are instructions for building the Science Pipelines.
+
+#. Check the `Science Pipelines changelog <https://lsst-dm.github.io/lsst_git_changelog/weekly/>`_ to make sure only the intended changes are on ``main``.
+   
+   If ``main`` contains only changes that are intended for the Science Pipelines build, proceed and build the main branch! Otherwise, a cherry-pick or backport branch should be built.
```

Comment:
My impression from talking through this with @isullivan was that, if you ended up backporting changes, you'd need a branch containing changes from each of them as you said. I'd love confirmation that that is the case, since I've not gone through this part of the process myself.